packages/protocol/test-sol/stability/SortedOracles.t.sol

```diff
@@ -0,0 +1,679 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.5.13;
+pragma experimental ABIEncoderV2;
+
+import { Test } from "celo-foundry/Test.sol";
+import { SortedOracles } from "../../contracts/stability/SortedOracles.sol";
+import "../../contracts/common/FixidityLib.sol";
+import "../../contracts/common/linkedlists/AddressSortedLinkedListWithMedian.sol";
+import "../../contracts/common/linkedlists/SortedLinkedListWithMedian.sol";
+
+contract SortedOraclesTest is Test {
+  using FixidityLib for FixidityLib.Fraction;
+  using AddressSortedLinkedListWithMedian for SortedLinkedListWithMedian.List;
+
+  SortedOracles sortedOracle;
+
+  address oracleAccount;
+  address aToken = 0x00000000000000000000000000000000DeaDBeef;
+  uint256 reportExpiry = 3600;
+
+  uint256 constant YEAR_IN_SECONDS = 31536000;
```

Comment:
https://github.com/celo-org/celo-monorepo/pull/10773/files#diff-c4eb9c7ae004c5b2108c0537376f13225f512e5acdf272c44f383b9a62c656e2

After my PR gets merge this could use the constant